### DISCUSSION TITLE
More Registry Keys!

### DISCUSSION THREAD

**Discussion by xM4ddy** [xM4ddy/OFGB#15] [Ideas]
## 👋 Welcome!
Here is a discussion to add useful registry keys that may be added to the program in future updates!

Please include the following

Use: What does it do? / Why Should It be Added To This App?
Source: Where you found the key(s) if applicable. A URL is fine!
Path: Something like `HKEY_CURRENT_USER\Software\Policies\Microsoft\Windows`
Key: Something like `DisableSearchBoxSuggestions`
Key Type: Something like `DWORD (32-bit)`
Value: What to set the value to. Something like `1 enables; 0 disables`

Template (add text between the ` `` `):
```
Use: ``
Source: ``
Path: ``
Key: ``
Key ``
Value: ``
```

**Comment by dmitriid** [comment]
Disabling web/bing search

This could be a useful feature to add: 

---

according to https://www.tomshardware.com/how-to/disable-windows-web-search

Path: `HKEY_CURRENT_USER\Software\Policies\Microsoft\Windows\Explorer` (create `Explorer` if it doesn't exist)

Key: DWORD (32-bit) `DisableSearchBoxSuggestions`

Value: `1`


---
And according to https://twitter.com/ultrawide219/status/1786104392753766466

Path: `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Search`

Key: DWORD (32-bit) `BingSearchEnabled`

Value: `0`

**Reply by xM4ddy** [reply]
Done https://github.com/xM4ddy/OFGB/commit/476000fccda60b7468e706b9d973c2cf060f37c1

**Comment by jboby93** [comment]
Disable Edge desktop search widget bar. I have this in a .reg file that I apply to new client machines when I set them up.  Works on Win10, presumably 11 as well.

Path: ```HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge```
Key: DWORD (32-bit) ```WebWidgetAllowed```
Value: ```0```

```
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge]
"WebWidgetAllowed"=dword:00000000
```

**Reply by xM4ddy** [reply]
Done https://github.com/xM4ddy/OFGB/commit/476000fccda60b7468e706b9d973c2cf060f37c1

**Comment by Aldaviva** [comment]
Use: `Disable "Notification Suggestions"` (the annoying notifications that periodically say "Turn off notifications from $app? We noticed you haven't opened these in a while.")
Source: [`https://superuser.com/a/1835257/339084`](https://superuser.com/a/1835257/339084) (I had to actually figure this one out myself because no one else could or did)
Path: `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Notifications\Settings\Windows.ActionCenter.SmartOptOut`
Key: `Enabled`
Key Type: `DWORD (32-bit)`
Value: `0 hides these notifications; 1 shows them (default)`

![image](https://github.com/xM4ddy/OFGB/assets/1417794/bb704312-7eb0-4f69-af82-886d2b22a9a4)

**Reply by xM4ddy** [reply]
Done https://github.com/xM4ddy/OFGB/commit/476000fccda60b7468e706b9d973c2cf060f37c1
